17thjan/Task_4.rb

```diff
@@ -0,0 +1,12 @@
+
+5.downto(1).each do |i|
```

Comment:
This logic created 10 Start in the first and last line, Please check your logic and modify accordingly